experiment/run-library.sh

```diff
@@ -675,6 +735,64 @@ function e() {
       "results.items.duration=$duration results.items.passed=$passed results.items.avg_duration=$avg_duration results.items.report_rc=$rc"
 }
 
+function ej() {
```

Comment:
Can't we just change `e()` for all the executions to work like this `ej()`? Do we need these two functions?

Reply:
I actually think that `ej()` is the most different function of all, because it needs to "grep" in JSON files and iterate over the list of "similar" items...

I guess that once we move more experiments to this "paradigm" we'll see new needs and we'll be able to factor more code :)